Kernel/Arch/aarch64/Exceptions.cpp

```diff
@@ -46,6 +46,10 @@ static void drop_to_el1()
     hypervisor_configuration_register_el2.RW = 1; // EL1 to use 64-bit mode
     Aarch64::HCR_EL2::write(hypervisor_configuration_register_el2);
 
+    // Setup initial exception stack
```

Comment:
Nit: "Setup" is a noun, "set up" (with space) is the verb. (Also in commit desc)

Reply:
Fixed :)